sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -280,6 +281,22 @@ export class Application extends RecordDataModel {
     },
   )
   supportingUsers?: SupportingUser[];
+  /**
+   * CRA income verifications associated with this application.
+   * The records are created once the student submits the application
+   * and the workflow is executed for the first time (Original Assessment)
+   * and also when supporting users submits its data.
+   */
+  @OneToMany(
+    () => CRAIncomeVerification,
+    (craIncomeVerification) => craIncomeVerification.application,
+    { eager: false, cascade: false, nullable: true },
+  )
+  @JoinColumn({
+    name: "application_id",
```

Comment:
please rename the name

Reply:
Sorry, I am not following, which name?

Reply:
`@JoinColumn({
    name: "application_id",`

Reply:
No, please ignore the comment. i got it

Reply:
The name is correct @ann-aot but you probably felt that it was wrong because the join was explicit for this one, but it could be removed. I will remove it since the other places does not have it explicit declared. The "application_id" information will come from the inverse relationship.
![image](https://user-images.githubusercontent.com/61259237/165583390-2408f637-f4b3-4c37-83b7-dcc1a6638efd.png)